test/unit/test_extension_wrapper_common.py

```diff
@@ -71,6 +73,30 @@ def test_bucketfs_credentials_default(mock_connect, filled_secrets):
     ))
```

Comment:
Maybe we should use 'mypassword' or so as constant to differentiate it from the query param

Reply:
I didn't understand this comment.

Reply:
in line 72, we have a line like this, one is the parameter name and one the value. I would suggest to change the value to mypassword

`        ['password'], ['password']`